src/mesh/RadioInterface.cpp

```diff
@@ -123,6 +123,20 @@ const RegionInfo regions[] = {
     */
     RDEF(MY_919, 919.0f, 924.0f, 100, 0, 27, true, true, false),
 
+    /*
+        Singapore
+        SG_868 Band 29: 868 - 869 MHz at 500mW, no restrictions.
+        https://www.imda.gov.sg/-/media/imda/files/regulation-licensing-and-consultations/ict-standards/telecommunication-standards/radio-comms/imdatssrd.pdf
+    */
+    RDEF(SG_868, 868.0f, 869.0f, 100, 0, 27, true, false, false),
```

Comment:
Band 29 frequency start is 866MHz.

<img width="980" alt="Screenshot 2024-02-02 23 19 11" src="https://github.com/meshtastic/firmware/assets/48680741/4e4a77d4-d53c-41ef-8669-3d88c5058751">

Reply:
The 868 band is slated for refarming and commercial auction per that document (see footnote 3 of p15)
I'm not clear on if that means they're still allowing licenseless use of that spectrum or merely allowing a grace period for migration to 923

Reply:
@andrekir @code8buster I’ve removed the 868MHz band from this PR and the protobufs PR which should resolve this thread.